src/org/mozilla/javascript/regexp/NativeRegExp.java

```diff
@@ -2512,14 +2548,7 @@ private static void reportError(String messageId, String arg) {
         throw ScriptRuntime.constructError("SyntaxError", msg);
     }
 
-    private static final int Id_lastIndex = 1,
-            Id_source = 2,
-            Id_flags = 3,
-            Id_global = 4,
-            Id_ignoreCase = 5,
-            Id_multiline = 6,
-            Id_sticky = 7,
-            MAX_INSTANCE_ID = 7;
+    private static final int Id_lastIndex = 1, Id_source = 2, MAX_INSTANCE_ID = Id_source;
```

Comment:
Sry, looks like `source` also ought to be an accessor property now:
https://tc39.es/ecma262/#sec-get-regexp.prototype.source
https://tc39.es/ecma262/#sec-get-regexp.prototype.source

Reply:
no problem, stay tuned :-)